engine/htmlhint/reporter.js

```diff
@@ -0,0 +1,54 @@
+const template = module.exports = {
```

Comment:
As I already suggested: it seems to be a base implementation for most of the reporters. Why not to create npm package for that?

Reply:
but how use with `lhexec` ?

Reply:
I propose this idea should be discussed in the separate issue